src/index.ts

```diff
@@ -82,19 +82,19 @@ export class Aracari<T extends HTMLElement = HTMLElement> {
   ) {
     let node;
     const { at, perserveWord, replacementIndex = 0 } = options;
+
     if (at) {
       node = this.getNodeByAddress(at);
     } else {
       node = this.getTextNode(text);
     }
+    const delimiter = perserveWord ? "\\b" : "";
```

Comment:
just wanted to confirm if `\\b` is what you wanted to use vs. `\b`. 

i was playing around with it here https://www.regular-expressions.info/javascriptexample.html. i was able to find a match using the single slash but wasn't successful with the double slash.

Reply:
Good question. We need the double slash because it is in string form then passed to `new ReqExp` which without the double slash escapes the `b`.